userspace/libscap/engine/source_plugin/source_plugin.c

```diff
@@ -206,10 +206,6 @@ static int32_t next(struct scap_engine_handle engine,
 	// Sanity checks in case a plugin implements a non-syscall event source.
 	// If a plugin has event sourcing capability and has a specific ID, then
 	// it is allowed to produce only plugin events of its own event source.
-	uint8_t* pplugin_id = (uint8_t*)evt + sizeof(scap_evt) + sizeof(uint32_t) + sizeof(uint32_t);
```

Comment:
Here, we don't yet know if this is a pluginevent, thus we cannot read nothing (we are not sure what we are reading)